app/lib/common/utils/constants.dart

```diff
@@ -57,3 +64,25 @@ const List<TargetPlatform> desktopPlatforms = [
 
 // hide bottom nav at locations.
 const List<String> hideNavLocations = ['/updates/post', '/updates/edit'];
```

Comment:
Not related to this PR but this constant needs to be removed. Nav bar can be hidden by using ```rootNavigatorKey``` (was not in knowledge at the time)